airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -12,9 +12,9 @@ java {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.28.20'
+    cdkVersionRequired = '0.28.22'
     features = ['db-sources', 'datastore-postgres']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
Have you tried publishing a new CDK version and then setting useLocalCdk = false? 

That's when it was failing for me w/ NoClassDefError issues

Reply:
I'll do so right now. I was waiting for the CI to go green.